image/templates/helm/shared/internal/scanner-v4-config-shape.yaml

```diff
@@ -0,0 +1,80 @@
+scannerV4:
+  disable: null # bool
+  indexer:
+    disable: null # bool
+    logLevel: null # string
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    replicas: null # int
+    autoscaling:
+      disable: null # bool
+      minReplicas: null # int
+      maxReplicas: null # int
+    nodeSelector: null # string | dict
+    tolerations: null # [dict]
+    affinity: null # dict
+    resources: null # string | dict
+    serviceTLS:
+      cert: null # string
+      key: null # string
+      generate: null # bool
+  matcher:
+    disable: null # bool
+    logLevel: null # string
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    replicas: null # int
+    autoscaling:
+      disable: null # bool
+      minReplicas: null # int
+      maxReplicas: null # int
+    nodeSelector: null # string | dict
+    tolerations: null # [dict]
+    affinity: null # dict
+    resources: null # string | dict
+    serviceTLS:
+      cert: null # string
+      key: null # string
+      generate: null # bool
+  db:
+    image:
+      registry: null # string
+      name: null # string
+      tag: null # string
+      fullRef: null # string
+    source:
```

Comment:
From what I can tell, this is only set in a configmap for Central if Central is connecting to a BYODB. I wonder if we hardcode it in the codebase otherwise? Worth looking into